java_archives.bzl

```diff
@@ -0,0 +1,62 @@
+load("//private/remote:temurin_archive.bzl", "temurin_archive")
+
+def repositories():
+    temurin_archive(
+        name = "temurin21_jre_amd64",
+        sha256 = "277f4084bee875f127a978253cfbaad09c08df597feaf5ccc82d2206962279a3",
+        strip_prefix = "jdk-21.0.1+12-jre",
+        urls = ["https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21.0.1+12/OpenJDK21U-jre_x64_linux_hotspot_21.0.1_12.tar.gz"],
+        version = "21.0.1+12",
```

Comment:
@mihaimaruseac specifically, do you know how to make this value (version) accessible in another build file somewhere?

Reply:
this is instead being exported to java/BUILD via JAVA_RELEASE_VERISONS